editor/editor_themes.cpp

```diff
@@ -724,14 +724,56 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_icon("visibility_xray", "PopupMenu", theme->get_icon("GuiVisibilityXray", "EditorIcons"));
 	theme->set_constant("vseparation", "PopupMenu", (extra_spacing + default_margin_size + 1) * EDSCALE);
 
-	Ref<StyleBoxFlat> sub_inspector_bg = make_flat_stylebox(dark_color_1.lerp(accent_color, 0.08), 2, 0, 2, 2);
-	sub_inspector_bg->set_border_width(SIDE_LEFT, 2);
-	sub_inspector_bg->set_border_width(SIDE_RIGHT, 2);
-	sub_inspector_bg->set_border_width(SIDE_BOTTOM, 2);
-	sub_inspector_bg->set_border_color(accent_color * Color(1, 1, 1, 0.3));
-	sub_inspector_bg->set_draw_center(true);
-
-	theme->set_stylebox("sub_inspector_bg", "Editor", sub_inspector_bg);
+	for (int i = 0; i < 16; i++) {
+		Color si_base_color = accent_color;
+
+		float hue_roate = (i * 2 % 16) / 16.0;
+		si_base_color.set_hsv(Math::fmod(float(si_base_color.get_h() + hue_roate), float(1.0)), si_base_color.get_s(), si_base_color.get_v());
+		si_base_color = accent_color.lerp(si_base_color, float(EDITOR_GET("docks/property_editor/subresource_hue_tint")));
+
+		Ref<StyleBoxFlat>
+				sub_inspector_bg = make_flat_stylebox(dark_color_1.lerp(si_base_color, 0.08), 2, 0, 2, 2);
```

Comment:
```suggestion
		Ref<StyleBoxFlat> sub_inspector_bg = make_flat_stylebox(dark_color_1.lerp(si_base_color, 0.08), 2, 0, 2, 2);
```

Reply:
clang format is doing this for some reason, I can't fix it. Guess I will rewrite the code so it does not.